TypeCobol.Analysis/Resource.Designer.cs

```diff
@@ -1,10 +1,10 @@
 ﻿//------------------------------------------------------------------------------
 // <auto-generated>
-//     This code was generated by a tool.
-//     Runtime Version:4.0.30319.42000
+//     Ce code a été généré par un outil.
```

Comment:
Strange the file is generated in French now ?

Reply:
Maybe it's because I have a french version of VS2019